src/api/application/application.ts

```diff
@@ -296,6 +305,16 @@ export class Application extends EmitterBase {
                ).then(() => undefined);
     }
 
+    /**
+     * Sets the zoom level of the application.
```

Comment:
Could we give a bit more info on what the settings mean? (0 is default, 1 is ..., -1 is ...)

Reply:
@tgoc99  updated docs